book/src/getting-started/5-cmake-integration.md

```diff
@@ -47,24 +59,35 @@ edition = "2021"
 ```
 
 We'll have to do multiple things:
-- Instruct cargo to create a staticlib
+- Instruct cargo to create a static library
 - Add `cxx`, `cxx-qt`, as well as `cxx-qt-lib` as dependencies
 - Add `cxx-qt-build` as a build dependency
 
-> Note that in order for cxx-qt to work, the `qmake` executable must be located. This is because cxx-qt relies on `qmake` to locate the necessary Qt libraries and header files on your system.
->
-> This will be done by setting `QMAKE` environment variable from CMake, so that Qt version found by CMake will be correctly passed down to Cargo.
+> If you've already followed the Cargo setup, most of this should already be done.
+> Make sure to change the `crate-type` to `"staticlib"` though!
 
 In the end, your `Cargo.toml` should look similar to this.
 
 ```toml,ignore
-{{#include ../../../examples/qml_minimal/rust/Cargo.toml:book_all}}
-```
+{{#include ../../../examples/qml_minimal/rust/Cargo.toml:book_static_lib}}
 
-> Note that instead of the `*.workspace = true` arguments for the CXX-Qt crates, you should instead use the versions from [crates.io](https://crates.io/search?q=cxx-qt).
-> As described in the code comment above each dependency.
+[dependencies]
+cxx = "1.0.95"
```

Comment:
we somehow need to remember to bump this CXX one ... Can you add a note next to the version in the workspace Cargo.toml ?